internal/mode/static/nginx/runtime/manager.go

```diff
@@ -29,6 +29,13 @@ type (
 
 var childProcPathFmt = "/proc/%[1]v/task/%[1]v/children"
 
+//go:generate go run github.com/maxbrunsfeld/counterfeiter/v6 . NginxPlusClient
+
+type NginxPlusClient interface {
```

Comment:
Since this is only used in this package, it can be lowercase (we don't need to export).

@kate-osborn Any thoughts on naming this interface? This is pretty straightforward though I know it's not the standard interface naming convention.

Reply:
In this case, I think the name is fine.

Reply:
great stuff